apps/proxy/entrypoint.sh

```
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -euo pipefail
```

Shellcheck (0.10.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)